### DISCUSSION TITLE
how to generate files for translators?

### DISCUSSION THREAD

**Discussion by kabrachups** [kazupon/vue-i18n#1653] [General]
I am working on Vue3 project and I want to generate files for translators to separate coding from translation. Something like we have in Django: _"Django then provides utilities to extract the translation strings into a [message file](https://docs.djangoproject.com/en/4.1/topics/i18n/#term-message-file). This file is a convenient way for translators to provide the equivalent of the translation strings in the target language. Once the translators have filled in the message file, it must be compiled. This process relies on the GNU gettext toolset."_

**Comment by samuelstroschein** [comment]
The upcoming inlang CLI could be leveraged to extract translations from code https://github.com/inlang/inlang.
